### PR TITLE
s390x: Change shortcut in SLE 15 for disk-format in YaST

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by okurz** [comment]
s/Changed/Change/ or even better "Adapt" :)

**Comment by okurz** [comment]
can you try to define the shortcut instead as a labeled %cmd entry instead? I think it's in lib/susedistribution.pm and you might need https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3649 at first which you can review and merge first after checking

**Comment by Soulofdestiny** [comment]
s/work/vacation/ - but thanks ;)

**Comment by Soulofdestiny** [comment]
I don't see how #3649 could be related to this PR
Can you clarify?

**Comment by okurz** [comment]
no, because defining the command hash based on the product version might work just because the susedistribution initialization happens later.

**Comment by Soulofdestiny** [comment]
verification run:
http://opeth.suse.de/tests/5677

**Comment by Soulofdestiny** [comment]
I would like to leave it as it is, because IMHO it's easier to read for people that are not familiar with s390x and especially this disk_activation dialog, since it's not present in other archs

File: tests/installation/disk_activation.pm [hunk 1]
```diff
@@ -94,9 +95,8 @@ sub run {
             send_key 'alt-a';                           # perform action button
             if (check_screen 'dasd-device-formatted') {
                 assert_screen 'action-list';
-                send_key 'f';
-                send_key 'f';                           # Pressing f twice because of bsc#940817
-                send_key 'ret';
+                # shortcut changed for sle 15
+                sle_version_at_least('15') ? send_key 'o' : send_key 'f';
```
Comment (okurz) [inline tests/installation/disk_activation.pm]: can you make that

```perl
send_key sle_version_at_least('15') ? 'o' : 'f';
```

Comment (Soulofdestiny) [reply tests/installation/disk_activation.pm]: sure, running verification again

Comment (Soulofdestiny) [reply tests/installation/disk_activation.pm]: http://opeth.suse.de/tests/5678

**Comment by Soulofdestiny** [comment]
updated

**Pull request merged**